types/licences.d.ts

```diff
@@ -43,9 +43,20 @@ export interface RoContactDetailsService {
   ) => Promise<ResponsibleOfficerAndContactDetailsResult>
 }
 
+interface PrisonerDetails {
+  firstName: string
+  lastname: string
+  offenderNo: stiring
+}
+
+interface Prisoner extends PrisonerDetails {
+  any
+}
```

Comment:
Not sure if this is typescript specific enough but the whole prisoner object looks quite extensive and contains another nested objects so wondering if this is ok to just make sure we have type safety on the prisoner parameters we require for the emails?

Reply:
this doesn't look like valid typescript to me?

Yes, its fine just to add the fields you are interested in.. not sure why you need the separate interface though?  Could you just have one?

Reply:
Thanks, corrected this in latest commit